examples/testsuite-content-tests/readme.md

```diff
@@ -483,6 +483,37 @@ if (jsonData?.service) {
   })};
 ```
 
+> **REMARK**:
+> If you have an API that returns a single value, such as a number or a boolean, instead of a JSON object you can set `value` to `.` to apply the test against the entire result
```

Comment:
Should this not be key instead of value?

> you can set `value` to `.` to apply

Reply:
Ugh, you are right.   Fixed in update.